<launch>
    <node pkg="joy_teleop_agv" type="joy_teleop_node.py" name="joy_teleop_node">
        
        <param name="joystick_port" value="js0"/>
        
        <param name="agv_max_linear_velocity" value="1.5"/>

        <param name="agv_max_angular_velocity" value="3.0"/>

    </node>
</launch>